devs/run-tests/24arraymap.ts

```diff
@@ -152,6 +152,49 @@ function testArrayAt() {
     assert(str.at(-6) === undefined, "arrayAtUndefined")
 }
 
+function localeCompareHelper(a: string, b: string) {
+    if (a < b) return -1
+    else if (a > b) return 1
+    else return 0
+}
+
+function testArraySort() {
+    msg("testArraySort")
+    const arr = [3, 1, 4, 1, 5, 9, 2, 6, 5, 3, 5]
+    arr.sort()
+    assert(arr.join() === "1,1,2,3,3,4,5,5,5,6,9", "sort1")
+
+    const numbers = [9, 3, 7, 1, 5]
+    numbers.sort((a, b) => a - b)
+    assert(numbers.join() === "1,3,5,7,9", "sort2")
+
+    const words = ["apple", "banana", "cherry", "date"]
+    words.sort()
+    assert(words.join() === "apple,banana,cherry,date", "sort3")
+
+    const people = [
+        { name: "Alice", age: 30 },
+        { name: "Bob", age: 25 },
+        { name: "Charlie", age: 35 },
+    ]
+    people.sort((a, b) => a.age - b.age)
+    assert(people.map(p => p.name).join() === "Bob,Alice,Charlie", "sort4")
+
+    const mixed = [5, "apple", 2, "banana", 9, "cherry"]
+    mixed.sort((a, b) => {
+        if (typeof a === "string" && typeof b === "string") {
+            localeCompareHelper(a, b)
```

Comment:
`return` missing? not sure why this isn't an error...

Reply:
It's an error in the tests. Not sure how I missed that. Maybe deleted after running the tests. 

Added the return now. Thank you.